.gitignore

```diff
@@ -1,4 +1,6 @@
-/vendor/
-/composer.lock
 /.phpcs-cache
 /.phpcs.xml
```

Comment:
Není tohle typo? Bývá to bez tečky

Reply:
tak jak jsi to poslal na gitlab tak jsem ti hodil sem :D ale jn, vypadá to na typo :) upravím pak